src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -238,6 +238,14 @@ public boolean walkToConstructionSite(final BlockPos currentBlock)
         return worker.isWorkerAtSiteWithMove(workFrom, STANDARD_WORKING_RANGE) || MathUtils.twoDimDistance(worker.getPosition(), workFrom) < MIN_WORKING_RANGE;
     }
 
+    /**
+     * Checks for blocks that need to be treated as deco
+     */
+    protected boolean isDecoItem(Block block)
+    {
+        return block instanceof CoralBlock || block instanceof LanternBlock || block instanceof BannerBlock || block instanceof WallSignBlock;
```

Comment:
Yeah no, this is now officially way to much OR-InstanceOf combos.
Add a config entry todo this, or add a simple list based registry in the api, regardless this is not a great maintainable approach anymore.

Reply:
Or add a Tag?
That would probably be the best approach.

Reply:
Yeah, a tag is probably the best thing.